lib/configuration.js

```diff
@@ -239,6 +240,22 @@ function applyConfiguration(inConfig) {
     throw new Error('Could not configure Dredd');
   }
 
+  // HTTP settings for the 'request' library. Currently cannot be customized
+  // by users, but in the future it should be. Leaking of 'request' library's
+  // public interface must be prevented though - sensible user options should
+  // be coerced to this object.
+  outConfig.http = {};
+
+  // Log information about the HTTP proxy settings
+  const proxySettings = getProxySettings(process.env);
+  if (proxySettings.length) {
```

Comment:
That's a good point. It is not a bug, but it might be convenient to have the logging to reflect the behavior. If the env vars resolution happened in Dredd itself, it might be simpler, but it's still just a few lines of code. I see this as another separate GitHub issue.

Reply:
https://github.com/apiaryio/dredd/issues/1309